Users/Install.rst

```diff
@@ -280,6 +280,14 @@ For x86 python or for x64 python:
 
 Replace "XXX" with the ClangSharp version you received from nuget.
 
+
+Integrating Shell Autocompletion
+--------------------------------
+
+You can also use coala for tab completion in bash or zsh. This will autocomplete coala arguments and bear names.
+You can set them up `here <http://coala.readthedocs.io/en/stable/Users/Tutorials/Shell_Autocompletion.html>`_.
+
```

Comment:
L287 & L288 have length ~ 112, so you try moving stuff from to new lines so that it stays in length limit ie (79)
😃  👍